modules/swagger-codegen/src/main/resources/aspnetcore/Dockerfile.mustache

```diff
@@ -1,11 +1,16 @@
-FROM microsoft/dotnet:1.0.3-sdk-projectjson
+FROM microsoft/aspnetcore-build:2.0 AS build-env
+WORKDIR /app
 
-ENV DOTNET_CLI_TELEMETRY_OPTOUT 1
```

Comment:
If there's not a specific reason for dropping this, could you please carry this over to the runtime build?

Removing is a breaking change and a potential privacy concern for existing users of the generator.